### DISCUSSION TITLE
isopleth phase diagrams?

### DISCUSSION THREAD

**Discussion by vahid2364** [pycalphad/pycalphad#483] [Q&A]
Hi,

Is there an example for plotting isopleth phase diagrams from multi-component databases?

Best regards,

**Comment by richardotis** [comment]
@bocklund may have a script that can approximate the isopleth, but it's more of a hack than a general solution.
We are working on a new mapping capability that will enable isopleth calculation, but I don't have have an estimated release date for it yet.
